packages/odyssey-react-mui/src/Field.tsx

```diff
@@ -31,6 +34,10 @@ export type FieldProps = {
    * If `error` is not undefined, the `input` will indicate an error.
    */
   errorMessage?: string;
+  /**
+   * Prop to render multiple level errors
+   */
+  errorMessages?: string[];
```

Comment:
Can you combine this into a single prop with a type union: `string | string []`

Reply:
👍  will check if Field is an internal only component